### PR TITLE
Default PUID and GUID to 2000

### PR DESCRIPTION
**Pull request by mmskv**
#### Summary
Change PUID and GUID in guide as mattermost/mattermost-docker sets PUID and GUID to 2000 by default in [app/Dockerfile](https://github.com/mattermost/mattermost-docker/blob/master/app/Dockerfile#L9)

I had an issue when starting the containers by this guide, specifically at the `chown` line in the set up. Changing directory group to `docker` did not help, but running `chown -R 2000:2000 ./volumes/app/mattermost/` did fix the permissions issue.

### CODE REVIEW

File: source/install/prod-docker.rst [hunk 1]
```diff
@@ -38,13 +38,22 @@ Docker Setup on Ubuntu
 
 3. **Deploy the Mattermost Docker** 
 
-You can get the :code:`uid` (user ID) and :code:`gid` (group ID) of the Docker user by running:
+Mattermost by default uses :code:`uid=2000` (user ID) and :code:`gid=2000` to access :code:`./volumes/app/`.
 
-.. code:: bash
-   
-   id <username>
+If you need to use another group or user IDs you can uncomment following lines in :code:`docker-compose.yml` (also change :code:`chown -R 2000:2000` in set-up commands to your :code:`uid:gid`)
```
Comment (justinegeffen) [inline source/install/prod-docker.rst:43]:
```suggestion
If you need to use another group or user IDs you can uncomment following lines in `docker-compose.yml` (also change `chown -R 2000:2000` in set-up commands to your `uid:gid`)
```

**Review by justinegeffen — COMMENTED** [review]
Just a formatting fix from my side. Thank you! :)

**Review by cwarnermm — APPROVED** [review]
Thank you for this update! Next steps include a technical review for accuracy.

**Comment by mmskv** [comment]
Just noticed that yesterday `mattermost/mattermost-docker` was [deprecated](https://github.com/mattermost/mattermost-docker/commit/f394fb1d15b02c7de89f9fe6debbdac740228c75) in favor of `mattermost/docker`, so the [installation documenation](https://docs.mattermost.com/install/prod-docker.html) and this PR are out of date now.

Should this documentation now point to https://github.com/mattermost/docker or should it be removed altogether?

**Comment by justinegeffen** [comment]
Great question, @mmskv! I've added @mrckndt as reviewer as he's working on the Docker migration/deprecation and is best-situated to answer that question. As the docs are in progress his review may be slightly delayed but we'll revert as soon as possible! Thank you for noting that. :)

**Review by mrckndt — APPROVED** [review]
Looks good to me

**Review by justinegeffen — APPROVED** [review]

**Comment by mmskv** [comment]
> Looks good to me

Uhmm what exactly?

Please see [this comment](https://github.com/mattermost/docs/pull/4767#issuecomment-887867409)

**Comment by mrckndt** [comment]
@mmskv The PR looks good. It's true the mattermost-docker repository is deprecated now but the UIDs and GIDs are still the same and the article will be updated to match the new repository. The articles itself will not be deleted but updated soon. So for me it's fine if we merge it for now.

**Comment by mmskv** [comment]
ok, I'm fine with the merge too

**Comment by justinegeffen** [comment]
Thanks, @mrckndt, appreciate your review and help! :)

**Pull request merged**